api/standard/trace.go

```diff
@@ -138,9 +138,15 @@ const (
 
 // Standard attribute keys for RPC.
 const (
-	// The RPC service name.
+	// A string identfying the remoting system.
```

Comment:
Looks like an error in the spec as well,

```suggestion
	// A string identifying the remote system.
```

Reply:
On second read, `remoting` is probably correct.

```suggestion
	// A string identifying the remoting system.
```